spec/mongoid/scopable_spec.rb

```diff
@@ -398,6 +398,41 @@ class << Band
         end
       end
 
+      context 'when the block is an none scope' do
+
+        context 'when there is no default scope' do
+
+          before do
+            Band.scope(:nothing, ->{ none })
```

Comment:
This will leak nothing scope for all following tests. Maybe it will be better to move this scope to model since it is used twice.